game_view.cpp

```diff
@@ -278,19 +278,20 @@ void game_view::draw_player_coords() noexcept
 {
     sf::Text text;
     text.setFont(m_game_resources.get_font());
+    text.setCharacterSize(24);
```

Comment:
Hi @EvoLandEco, I appreciate this minor improvement and I will accept it. However, keep in mind that commits should ideally do exactly one thing. There are exceptions (it is called 'The Boy Scout Rule', where you leave the code a little cleaner than you found it, which usually applies to indentation). Here, however, you change the **behavior** of the program, and this may result in merge conflict with people that work on that text. 

But again, good stuff!

Reply:
I just got to know that my subsequent commits will be added to this pull request, so I decided to solve several issues in one PR, but of course I will try to avoid it next time